model/config.go

```diff
@@ -1383,6 +1389,10 @@ func (o *Config) IsValid() *AppError {
 		return NewLocAppError("Config.IsValid", "model.config.is_valid.file_salt.app_error", nil, "")
 	}
 
+	if len(*o.FileSettings.AtRestEncryptKey) > 0 && len(*o.FileSettings.AtRestEncryptKey) < 32 {
```

Comment:
we're not.  It must be either 0 or >= 32.  0 = do not encrypt.  I required at least 32 because that's what was required for the at rest encryption key for the database.